<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-IFDRR3S" failures="2" tests="2" name="Test4" time="30.545" errors="0" timestamp="2021-08-31T23:53:26 IST">
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase4" time="10.869">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.Sample_Prep.PreparationCodes.SampleTestNG.testcase4(SampleTestNG.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase4" time="18.004">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.Sample_Prep.PreparationCodes.SampleTestNG.testcase4(SampleTestNG.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase2" time="0.0">
    <ignored/>
  </testcase> <!-- testcase2 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase3" time="0.0">
    <ignored/>
  </testcase> <!-- testcase3 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase1" time="0.0">
    <ignored/>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Test4 -->
